WordPress/JetpackStatsWidgets/Views/SingleStatView.swift

```diff
@@ -1,19 +1,59 @@
 import SwiftUI
+import WidgetKit
 
 struct SingleStatView: View {
+    let title: String
+    let description: String
+    let valueTitle: String
+    let value: Int
 
-    let viewData: GroupedViewData
+    @Environment(\.showsWidgetContainerBackground) var showsWidgetContainerBackground: Bool
+
+    init(viewData: GroupedViewData) {
+        self.title = viewData.widgetTitle
+        self.description = viewData.siteName
+        self.valueTitle = viewData.upperLeftTitle
+        self.value = viewData.upperLeftValue
+    }
```

Comment:
Did you consider delegating to `self.init(title:...`?

Reply:
Good observation. I'll use `self.init..`